app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
That isn't a documented api in katello, nor do i think it should be (as its determined by candlepin/rhsm) and involves weird non-katello terminology.  While I'm not the biggest fan of having this api, asking users to use that api is not great.